app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -244,6 +244,7 @@ class HistoryFragment : LibraryPageFragment<History>(), UserInteractionHandler {
             true
         }
         R.id.history_search -> {
+            requireComponents.analytics.metrics.track(Event.HistorySearchIconTapped)
```

Comment:
would it make sense to move this to the interactor so this `metrics.track()` call is covered in a unit test?

Reply:
True, that is also a solution.  However, since the metrics I'm tracking is `HistorySearchIconTapped`  I felt like if I put it in the interactor then it's not really tracking that anymore.  Currently it will be the same thing but if anyone re-used the interactor then it will be tracking too many events.